pkgs/tools/misc/kronometer/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  kdeDerivation, kdeWrapper, fetchurl, lib,
+  ecm, kdoctools,
+  kconfig, kinit
+}:
+
+let
+  unwrapped = let
+    pname = "kronometer";
+    version = "2.1.0";
+  in kdeDerivation rec {
+    name = "${pname}-${version}";
```

Comment:
I've pushed a new set of changes that fixes the license - sloppy copy paste from here.

As for defining the variables via ```let``` instead of just passing them to the derivation, is there a preference or SingleRightWay(tm)?